src/libsyncengine/jobs/network/abstracttokennetworkjob.h

```diff
@@ -33,7 +33,7 @@ class AbstractTokenNetworkJob : public AbstractNetworkJob {
         };
 
     public:
-        typedef enum { ApiDrive, ApiDriveByUser, ApiProfile, ApiNotifyDrive } ApiType;
+        typedef enum { ApiDrive, ApiDriveByUser, ApiProfile, ApiNotifyDrive, ApiDesktop } ApiType;
```

Comment:
Would it be a lot of extra changes if this enum becomes a `enum class`?

Reply:
Not a lot, but I plan to come across all the `typedef enum` soon. I will do it a this occasion if it sound good to you? https://infomaniak.atlassian.net/browse/KDESKTOP-917